sdk/videoanalyzer/video-analyzer-edge/samples/javascript/README.md

```diff
@@ -0,0 +1,32 @@
+# Azure Video Analyzer Edge client library for JavaScript
```

Comment:
Really this readme should look more like here:

https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/tables/data-tables/samples/v12/typescript

But rather than having to hand-author it, you can generate it with a tool, see this example: https://github.com/Azure/azure-sdk-for-js/pull/14799/files

Reply:
Yes we will have this done for GA